src/test/resources/fixtures/cmd/commands/delete-operation/asyncapi-2/delete-operation.before.json

```diff
@@ -8,7 +8,10 @@
     "myChannel": {
       "description": "This is myChannel description",
       "subscribe": {
-        "summary": "Subscribe summary"
+        "summary": "Subscribe summary",
+        "message": {
+          "name": "bullshit"
```

Comment:
Can we use less offensive test data?  :)

Reply:
Sorry, copied from the issue test data.